src/main/java/org/openrewrite/java/migrate/joda/JodaTimeRecipe.java

```diff
@@ -15,17 +15,19 @@
  */
 package org.openrewrite.java.migrate.joda;
 
+import lombok.Getter;
 import org.openrewrite.ExecutionContext;
 import org.openrewrite.ScanningRecipe;
+import org.openrewrite.java.tree.J;
 import org.openrewrite.java.tree.J.VariableDeclarations.NamedVariable;
+import org.openrewrite.java.tree.JavaType;
 
-import java.util.HashSet;
-import java.util.Set;
+import java.util.*;
 
-public class JodaTimeRecipe extends ScanningRecipe<Set<NamedVariable>> {
+public class JodaTimeRecipe extends ScanningRecipe<JodaTimeRecipe.Accumulator> {
     @Override
     public String getDisplayName() {
-        return "Migrate Joda Time to Java Time";
+        return "Migrate joda time to java time";
```

Comment:
Some capitalization is actually fine here; just not every word.
```suggestion
        return "Migrate Joda-Time to Java time";
```